.vscode/settings.json

```diff
@@ -21,6 +21,7 @@
   "prettier.bracketSpacing": false,
   "prettier.singleQuote": true,
   "prettier.printWidth": 80,
+  "prettier.trailingComma": "all",
```

Comment:
Can we set it to `es5`? See https://github.com/prettier/prettier#trailing-commas.